### ISSUE TITLE
R Study group 2018 planning session 

### ISSUE THREAD

**Issue by ttimbers** [minisciencegirl/studyGroup#195]
Curious about the R study group (https://ubc-r-study-group.github.io/studyGroup/)? Want to learn R/ Python and don't know where to start? Have a cool package/ tool that you want to share with the community?

Come to the first study group meetup of 2018 Fall! </br>
**Time:** Wednesday, Sept 12, 12:30-1:30PM
**Place:** Koerner Library (Room 216)

We will talk about topics YOU are interested in and schedule the fall sessions. Come help us plan out this year and sign up for a GitHub account if you don't have one.

In the past, we have run various utility Belt sessions, which are aimed at beginners (and will cover various R packages in tidyverse on data manipulation and visualization). If there is interest, we will also work through Hadley Wickham's Advanced R book http://adv-r.had.co.nz/. 

We would also like to hear from the community the topics they are interested in. So come on out and share your ideas with us.

Hope to see you all there!

The R Study Group Organizing Team

**Comment by ahlyon** [comment]
Hi all,

I’ll try to make it tomorrow but unfortunately my lab group is having a weekly meeting from 11:30 to 1 this semester. Wed. 12:30 – 1:30 be the regular time slot? If so I’ll try to see if I can negotiate leaving lab meeting early as I would really like keep coming regularly!

**Comment by ttimbers** [comment]
Yes, 12:30-1:30 will be the regular timeslot this fall. Come by even if you can make it just for the second half!

**Comment by ahlyon** [comment]
OK thanks! I will try to figure out a way to come anyway. I won’t be able to make it today but would like to request an intro session on GitHub. Thank you!

**Comment by minisciencegirl** [comment]
Hey @ahlyon: we've actually shifted the meeting time to 1PM - 2PM on Wednesdays! Hope to see you at the next session!

**Comment by ahlyon** [comment]
Fantastic!

**Comment by uyennn** [comment]
Hi,

I was unable to make the meeting today due to a special training session at my institute (BCCHRI). However, I will be able to join every other Wednesdays 1-2PM. :)
I would like to learn more about dimension reduction techniques such as t-SNE and PCA, and how to perform them in R. Thank you!

**Comment by ahlyon** [comment]
Hi all,

Just curious - has a list of upcoming topics been posted yet? Thanks!

**Comment by minisciencegirl** [comment]
Hi @ahlyon: 

Our new list of topics are listed here: https://github.com/UBC-R-Study-group/studyGroup/issues/1
By the way, we've moved ourselves over to a new site, please watch this repo: https://ubc-r-study-group.github.io/studyGroup/

Thanks!
Amy

**Comment by Acheron-37** [comment]
Hi all,
I've been  totally unable to attend the meetings at UBC, yet I want to ask whether  it is possible to continue the study grup activities or further meetings on GitHub. 
My thanks.
Caner
